src/tests/tests.pyproj

```diff
@@ -6,7 +6,7 @@
     <ProjectGuid>{250c535c-c060-4f0c-bd80-41f2bf373565}</ProjectGuid>
     <ProjectHome />
     <StartupFile>runtests.py</StartupFile>
-    <SearchPath />
+    <SearchPath>..\..\</SearchPath>
```

Comment:
This adds root folder of pythonnet to search paths. Helps when building with --inplace option.